config/jobs/kubernetes/sig-testing/conformance-e2e.yaml

```diff
@@ -88,7 +88,48 @@ periodics:
     description: Runs conformance tests using image against latest kubernetes master with a kubernetes-in-docker cluster
     testgrid-alert-email: davanum@gmail.com
     testgrid-num-columns-recent: '3'
-- interval: 1h
+- interval: 3h
+  name: ci-kubernetes-conformance-image-test-1-15-test
```

Comment:
```suggestion
  name: ci-kubernetes-conformance-image-1-15-test
```

Reply:
/shrug sure, I just followed the existing pattern

Reply:
No, that extra `test-` is not in the 1-14 test on L133